### ISSUE TITLE
Feature: iOS Live Photos

### ISSUE THREAD

**Issue by m4rkv5** [immich-app/immich#160]
iPhone Live photos are converted into images. If Immich is used as a backup, the old status cannot be restored.

**Comment by JM-Lemmi** [comment]
It would be nice to have compatibility with Samsung Motion Photos and the Google Format too.

They are implemented a bit differently from Apple. Instead of a MOV file next to it, it has a mp4 integrated into the jpg/heic file. [This repo](https://github.com/joemck/ExtractMotionPhotos) has a good deconstruction of the format.

**Comment by evulhotdog** [comment]
Excited to see this happen, this is the last major thing that I feel I'm missing to migrate from Google Photos!

**Comment by fazzer4x** [comment]
Thanks for the update! The live fotos feature is what I'm currently waiting on before making the jump.

**Comment by fazzer4x** [comment]
@meichthys Why would you want to destruct the fotos?

**Comment by meichthys** [comment]
Afaik we need to destruct the file to get to the video. I see that it could still be important to be able to reconstruct the file if we want to restore from back up, but just for viewing the files, it probably would not be necessary.

**Comment by m4rkv5** [comment]
Most other iOS apps I know save two files and view it as one in the app.

**Comment by meichthys** [comment]
Yes. That sounds right to me. The photo and live video are linked with a uuid.

**Comment by m4rkv5** [comment]
The playback animation style gets lost when I export the Live Photo via apple photos on macOs and reimport it.

**Comment by fazzer4x** [comment]
Digging through the-photo-gallery I noticed that imagor is using libvips to work with heic fotos. They in turn are using https://github.com/strukturag/libheif for de-/encoding. This library seems to support avif as well. But I'm not to sure about further details.
